Core/BotController.cs

```diff
@@ -192,6 +200,9 @@ public void ToggleBotStatus()
                 else
                 {
                     actionThread.Active = false;
+                    this.GrindSession.StopBotSession();
```

Comment:
What happens to a `GrindSession` when the application does not exits gracefully? and this method and this
```
this.GrindSession.StopBotSession();
```
does not called?

Reply:
it will record nothing for that session.

Reply:
okay just good to know